packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/oktatookta/appatidp.md

```diff
@@ -1,19 +1,29 @@
-When you are configuring federation between two Okta orgs using OpenID Connect:
+> **Note**: This guide will refer to the Okta org that represents the Identity Provider as **Org A** and the second Okta org as **Org B**.
 
-1. At the Okta org that represents the Identity Provider, select **Applications** and then click **Add Application**.
+1. At **Org A**, select **Applications** and then click **Add Application**.
```

Comment:
It is difficult to do something like "Org A" and "Org B" since we don't know where the reader is going to come in from to read our content. So we can't assume that the reader is going to start from the beginning of the guide, unfortunately, and referring to Org B or Org A without any context is more confusing than just saying "The Okta org you are using as the Identity Provider" or the just the "Okta org"....

Reply:
Got it, thanks for the feedback!